src/Language/Haskell/Liquid/Types/Types.hs

```diff
@@ -795,15 +796,19 @@ mapTyVarValue :: (tv1 -> tv2) -> RTVar tv1 s -> RTVar tv2 s
 mapTyVarValue f v = v {ty_var_value = f $ ty_var_value v}
 
 dropTyVarInfo :: RTVar tv s1 -> RTVar tv s2
-dropTyVarInfo v = v{ty_var_info = RTVNoInfo}
+dropTyVarInfo v = v{ty_var_info = RTVNoInfo True }
 
 data RTVInfo s
-  = RTVNoInfo
+  = RTVNoInfo { rtv_is_pol :: Bool }
   | RTVInfo { rtv_name   :: Symbol
             , rtv_kind   :: s
             , rtv_is_val :: Bool
+            , rtv_is_pol :: Bool 
```

Comment:
Can you add some documentation here to explain what this field means?eg an example of when the field is true/false?

Reply:
added documentation!